compute/cloud-client/src/main/java/compute/Quickstart.java

```diff
@@ -0,0 +1,216 @@
+      System.out.println("Listing instances ! ! ");
+      for (Instance zoneInstance : instancesClient.list(project, zone).iterateAll()) {
+        System.out.println(zoneInstance.getName());
+      }
+    }
+  }
+
+  //  [END compute_instances_list]
+
+  // [START compute_instances_list_all]
+
+  /**
+   * List the compute instances in the given project
+   *
+   * @param project Google Cloud Project id
+   */
+
+  public void listAllInstances(String project) throws IOException {
+    try (InstancesClient instancesClient = InstancesClient.create()) {
+      // Listing all instances for the project
+      AggregatedListPagedResponse response = instancesClient.aggregatedList(project);
+      System.out.println("Listing instances ! ! ");
+      for (Entry<String, InstancesScopedList> zoneInstances : response.iterateAll()) {
+        // Scoped by each zone; Check if instances in a zone is null, else iterate and list them
+        String zone = zoneInstances.getKey();
+        if (zoneInstances.getValue().getInstancesList() != null) {
+          for (Instance instance : zoneInstances.getValue().getInstancesList()) {
+            // getZone() returns the fully qualified address.
+            // Hence, strip it to get the zone name only
+            System.out
+                .println(instance.getName() + " at " + zone.substring(zone.lastIndexOf('/') + 1));
+          }
+        }
+      }
+    }
+  }
+
+  // [END compute_instances_list_all]
+
+  // [START compute_instances_delete]
+
+  /**
+   * Deletes the GCP compute instance specified by the instanceName param
+   *
+   * @param project      Google Cloud Project id
+   * @param zone         Google Cloud Project zone
+   * @param instanceName Name of the instance to be deleted
+   */
+  public void deleteInstance(String project, String zone, String instanceName)
+      throws IOException, InterruptedException {
+    try (InstancesClient instancesClient = InstancesClient.create()) {
+
+      // Set the properties of the instance which is to be deleted
+      Operation response = instancesClient.delete(project, zone, instanceName);
+
+      // [START compute_instances_operation_check]
+      ZoneOperationsClient zoneOperationClient = ZoneOperationsClient.create();
+      // Waits until the delete operation is completed
+      while (response.getStatus() != Status.DONE) {
+        response = zoneOperationClient.get(project, zone, response.getId());
+        if (response.getStatus() != Status.DONE) {
+          System.out.println("Deletion in process...");
+        }
+      }
+      // [END compute_instances_operation_check]
+
+      if (response.getError().toString() == "") {
+        System.out.println("Instance deleted successfully ! ! ");
+      } else {
+        System.out.println("Failed to delete instance ! ! " + response.getError());
+      }
+    }
+  }
+  // [END compute_instances_delete]
+
+
+  public void quickstart() throws IOException, InterruptedException {
```

Comment:
This quickstart doesn't look like it's actually being reference with region tags, so consider just removing it.